.github/workflows/msys2-htslib-build.yml

```diff
@@ -1,6 +1,8 @@
 name: Build a statically linked htslib.dll under msys2
 on:
   push:
+    branches:
+      - main
```

Comment:
This means we won't be able to test the `build` job on branches. I'd prefer to instead add `github.ref == 'refs/heads/main'` to the `if:` for the `release` job

Reply:
We can by manually invoking the workflow.

Reply:
Ah, ok. That makes sense. I forgot that TileDB-Inc devs prefer internal branches